Core/wbDefinitionsSF1.pas

```diff
@@ -9311,7 +9330,11 @@ procedure DefineSF1b;
         ], []),
         //BGSSpaceshipWeaponBindings_Component
         wbRStruct('Component Data', [
-          wbUnknown(SHWB)
+          wbStruct(SHWB, 'Ship Weapon Binding', [
+            wbInteger('Weapon Slot 1', itS32),
+            wbInteger('Weapon Slot 2', itS32),
+            wbInteger('Weapon Slot 3', itS32)
+          ])
```

Comment:
what exactly do these integers represent?

Reply:
integers relate to the ID of the item from the components list